configurator/project/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html>
+  <head>
+    <script type="module" src="./my-element.js"></script>
+  </head>
+  <body>
+    <div>Here are the contents:</div>
+    <my-element myNumber="5">
+      <div>Element is not upgraded</div>
```

Comment:
I'd suggest we remove this and simplify the example. It was included originally to show the full lifecycle of an element, but I think obscured the demo now.

Reply:
Done, simplified to a "Hello World" example.